test/TestServerProjectsLowLevel/paging/Generated/PagingClient.cs

```diff
@@ -1079,8 +1080,7 @@ IEnumerable<Page<BinaryData>> CreateEnumerable(string nextLink, int? pageSizeHin
         public virtual AsyncPageable<BinaryData> GetMultiplePagesFragmentNextLinkAsync(string tenant, string apiVersion, RequestContext context = null)
 #pragma warning restore AZC0002
         {
-            Argument.AssertNotNull(tenant, nameof(tenant));
-            Argument.AssertNotNull(apiVersion, nameof(apiVersion));
```

Comment:
We want to always be sending `apiVersion` in requests, so I think it makes sense to validate `apiVersion` here.

Reply:
Hm... that is an interesting place you have found! I'd expect it to use _apiVersion field, but for some reason it is generated as a method parameter. Need to debug this one.

Reply:
Ok, so these parameters don't have `origin: modelerfour:synthesized/api-version`, e.g.: https://github.com/Azure/autorest.csharp/blob/feature/v3/test/TestServerProjectsLowLevel/paging/Generated/CodeModel.yaml#L1739
(compare to https://github.com/Azure/autorest.csharp/blob/feature/v3/test/TestServerProjectsLowLevel/body-complex/Generated/CodeModel.yaml#L1972)

@timotheeguerin , do you know why these ones aren't marked as api-version?